README.md

```diff
@@ -116,6 +117,28 @@ public static void bubbleSort(int[] arr) {
 
 [⬆ back to top](#table-of-contents)
 
+### Binary Search
+```java
+  public static int binarySearch(int[] arr, int item) {
+    int low = 0;
+    int high = arr.length - 1;
+
+    while(low <= high) {
+      int mid = (low + high) / 2;
+      int cur_guess = arr[mid];
+      if(cur_guess == item) {
+        return mid;
+      } else if(cur_guess > item) {
+        high = mid - 1;
+      } else {
+        low = mid + 1;
+      }
+    }
+    return -1;
```

Comment:
Why does this return -1? It should return the index of the value for which you are searching.

Reply:
Because the value wasn't found in the array.